contracts/src/Rollup.sol

```diff
@@ -210,7 +229,53 @@ contract Rollup is RollupBase {
     }
 
     /// @inheritdoc IRollup
-    function stake() external payable override {
+    function addValidatorsToWhitelist(address[] calldata validators) external override onlyRole(DEFAULT_ADMIN_ROLE) {
+        for (uint256 i = 0; i < validators.length; i++) {
+            if (hasRole(VALIDATOR_WHITELIST_ROLE, validators[i])) {
```

Comment:
We can simplify this check in the next OpenZeppelin version which returns a bool for grant/RevokeRole() to indicate whether the call succeeds.